hedera-node/hedera-addressbook-service-impl/build.gradle.kts

```diff
@@ -36,5 +36,6 @@ testModuleInfo {
     requires("org.junit.jupiter.api")
     requires("org.mockito")
     requires("org.mockito.junit.jupiter")
+    requires("com.swirlds.common")
```

Comment:
Is this for a new publish group? I didn't think we were publishing anything new to com.swirlds.*